front/src/TweetForm.tsx

```diff
@@ -1,21 +1,36 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import Box from '@mui/material/Box';
 import TextField from '@mui/material/TextField';
 import Button from '@mui/material/Button';
 import useCallApi from './utils/api';
+import { TweetType } from './utils/types';
 
-type TweetFormProps = { getAndSetTweets: () => void };
+type TweetFormProps = { afterPostTweet: () => void; replySourceTweet: null | TweetType };
 
 const TweetForm: React.FC<TweetFormProps> = (props) => {
-    const [content, setContent] = useState<string>('');
+    const [content, setContent] = useState<string>(
+        props.replySourceTweet ? '@' + props.replySourceTweet?.user.username : ''
+    );
```

Comment:
ここはstateじゃなくて良い（じゃないほうが良い）
```suggestion
    cons content = props.replySourceTweet ? '@' + props.replySourceTweet.user.username : '';
```
たんにこれでok